tfjs-backend-webgpu/src/kernels/from_pixels_webgpu.ts

```diff
@@ -26,35 +26,144 @@ export class FromPixelsProgram implements WebGPUProgram {
   variableNames = ['A'];
```

Comment:
['A'] is not needed ?

Reply:
Done. And by assign variableNames to [], we don't need the change in shader_preprocessor file.